### PR TITLE
Cleaning

### PR DESCRIPTION
**Pull request by jburel**
Nothing to test just unifying the doc with the rest of the guide
cc @dominikl @pwalczysko

### CODE REVIEW

File: docs/r_walkthrough.rst [hunk 1]
```diff
@@ -57,36 +57,28 @@ Note: Everything we are going to do there can be done in Rstudio as
...
+Resources
+---------
 
 We will use data from the Image Data Resource (IDR).
 
--  IDR data \ https://idr.openmicroscopy.org/webclient/?show=project-51
+-  IDR data `idr0021 <https://idr.openmicroscopy.org/webclient/?show=project-51>`_.
 
--  Notebook \ https://github.com/ome/training-notebooks/blob/master/R/idr0021%20-%20Segmentation.ipynb
+-  Notebook :file:`notebooks/idr0021_Segmentation.ipynb`.
```
Comment (pwalczysko) [inline docs/r_walkthrough.rst]: this is not clickable and downloadable for me. Builds just as a simple text.
Screen Shot 2020-07-09 at 17 52 49

**Comment by pwalczysko** [comment]
One problem with the [notebook accessibulity](https://github.com/ome/omero-guide-r/pull/6#discussion_r452357260) Otherwise looks fine

File: docs/r_walkthrough.rst (hunk 1 above)
Comment (jburel) [reply docs/r_walkthrough.rst]: it is a simple text.

**Comment by dominikl** [comment]
Looks good, thanks @jburel 👍

File: docs/r_walkthrough.rst (hunk 1 above)
Comment (pwalczysko) [reply docs/r_walkthrough.rst]: Aha, okay.

Comment (jburel) [reply docs/r_walkthrough.rst]: After second though, I will link to the notebook in mybinder.

**Comment by jburel** [comment]
ok to merge?

**Comment by pwalczysko** [comment]
Yes, looks fine, ready to merge fmpov

**Pull request merged**